src/test/java/edu/kit/iti/formal/pse2018/control/SupervisorControlTest.java

```diff
@@ -0,0 +1,191 @@
+package edu.kit.iti.formal.pse2018.control;
+
+import edu.kit.iti.formal.pse2018.evote.model.SupervisorControlToModelIF;
+import edu.kit.iti.formal.pse2018.evote.utils.ElectionDataIF;
+import edu.kit.iti.formal.pse2018.evote.view.SupervisorControlToViewIF;
+import edu.kit.iti.formal.pse2018.evote.control.supervisorcontrol.SupervisorControl;
+import org.junit.Test;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import static org.junit.Assert.assertEquals;
+
+public class SupervisorControlTest {
+	@Test
+	public void auth() {
+		View view = new View();

```

Comment:
Use mocking with Mockito instead of private classes, see other tests for examples